stemcell_builder/stages/bosh_ntpd/assets/ntp.conf

```diff
@@ -0,0 +1,13 @@
+driftfile /var/lib/ntp/ntp.drift
+
+statistics loopstats peerstats clockstats
+filegen loopstats file loopstats type day enable
+filegen peerstats file peerstats type day enable
+filegen clockstats file clockstats type day enable
+
+restrict -4 default kod notrap nomodify nopeer noquery
+restrict -6 default kod notrap nomodify nopeer noquery
+
+restrict 127.0.0.1
```

Comment:
i thought you guys mentioned that restrict 127... causes ntpd to fail actually fetching time from remote servers?